### ISSUE TITLE
Why "web-based & hosted and with a gratis plan" only

### ISSUE THREAD

**Issue by hugoroy** [libreprojects/libreprojects#111]
I would like also to send people to hosted and web based "libreprojects" even if they are not gratis. Maybe a special filter could be added to also show paid-for services?

**Comment by yuwash** [comment]
You could easily install any FLOSS web app if you’re willing to pay for hosting. Thus the difference to already existing general FLOSS directories would diminish. Anyway libreprojects is FLOSS too, so you can just fork it and start your own site to showcase any kind of sites using this software.

**Comment by hugoroy** [comment]
1. Not all web apps are "easily install"; hosting per se is not necessarily the biggest cost. Anyway this argument also works for gratis services so I don't see how this is relevant with my question.
2. Which directories do you refer to?
3. I don't want to fork Libre Projects.

**Comment by jancborchardt** [comment]
Yeah, good point. I’ve been thinking about this as well cause otherwise it ties »free and open source software« too much to the idea that it’s free as in not needing to pay money. Cause paying money for open source should be encouraged.

If you have proposals of services like that which should be added, please list them or ideally directly make pull requests. :)

**Comment by ironbishop** [comment]
It's good to promote non-gratis free softwares. However, I would prefer that the gratis projects had greater importance than paid ones, here. In addition, many projects, such as Wordpress, offers both gratis and paid services: how do we manage them?

**Comment by hugoroy** [comment]
Just use whatever filters you have already and add one to differentiate between paid-for and totally gratis. :-)
